Cargo.toml

```diff
@@ -23,6 +23,7 @@ libm = { version = "0.2.0", optional = true }
 default = ["std"]
 std = []
 i128 = []
+copysign = []
```

Comment:
Do we really need a feature for this? I'd rather leave it to `autocfg`, unless there's a strong reason to expose that condition.

Reply:
Yes, because `autocfg`  is not something that can be used on custom targets (related https://github.com/rust-lang/cargo/issues/7501). I would go as far as saying that `autocfg` is an anti-pattern for supporting custom targets currently, the way probing capabilities currently works is very incompatible if your project requires being built in a specialised way. One example a project that needs `num-traits` and a custom target is `rust-gpu`.

Reply:
Yeah, even as the `autocfg` author I am frustrated by that situation, but I haven't found time to design/push for tool changes that would alleviate it. I think having something like `TARGET_RUSTFLAGS` from cargo set for build scripts would solve a lot of issues. Might you be interested in pursuing this, since it impacts targets you work with?

Reply:
FWIW, `autocfg` uses `CARGO_ENCODED_RUSTFLAGS` now, which should be more reliable.

If you're not even happy with that, then I'd prefer a plain version check rather than a feature for every little implementation detail. ("i128" is a little different because it actually adds to the API.)